README.md

```diff
@@ -146,10 +161,20 @@ query ($id: ID!) {
 }
 ```
 
-- @external - This is covered in the tests above.
+- `@external` - This is covered in the tests above.
 - `extends` or `@extends` - This is covered in the `products` subgraph extension of the `User`
 - `ftv1` (Federated Traces version 1) - A query with the `apollo-federated-include-trace:ftv1` header will be sent to the `products` subgraph which should return a value for the `extensions.ftv1` in the result.
   - _NOTE: In the initial release of this testing strategy, we will not be validating `ftv1` to ensure it's in the proper format_
+- `@link`
+  - Must be seen as a valid schema directive in subgraph library schemas. Is verified by checking for its inclusion in the `query { _service { sdl } }` result.
+- `@tag`
+  - Must be seen as a valid schema directive in subgraph library schemas. Is verified by checking for its inclusion in the `query { _service { sdl } }` result.
+- `@shareable`
+  - Must be seen as a valid schema directive in subgraph library schemas. Is verified by checking for its inclusion in the `query { _service { sdl } }` result.
+- `@override`
+  - Must be seen as a valid schema directive in subgraph library schemas. Is verified by checking for its inclusion in the `query { _service { sdl } }` result.
+- `@inaccessible`
+  - Must be seen as a valid schema directive in subgraph library schemas. Is verified by checking for its inclusion in the `query { _service { sdl } }` result.
```

Comment:
Thoughts 💭 

I'd love to find a terminology to distinguish between:

1. the subgraph schema that written by users and is input to subgraph libraries
2. the subgraph schema that is output by schemas library

I usually refer to 2. as the `service sdl` but maybe I'm the only one 😅 . I don't really have a solution so feel free to ignore this comment altogether

Reply:
"2" is "federated introspection", usually

Reply:
Interestingly, `federation-jvm` has "federated" [test fixtures](https://github.com/apollographql/federation-jvm/tree/master/graphql-java-support/src/test/resources/com/apollographql/federation/graphqljava/schemas) that do not match 1:1 with 2 in Fed1. 

Fed2 makes the "federated" test fixtures match 1:1 with the "service" ones so that's something.

All in all, I like the "federated introspection" albeit the "federated" might hint at a "supergraph" thing. Naming is hard 😅

Reply:
Great point and I like your terminology. I'll adjust this (and update the descriptions in the README for some of those directives to also mention that we don't just test for _service sdl inclusion, that we do test functionality). Thanks!